modules/accredited_representative_portal/app/serializers/accredited_representative_portal/power_of_attorney_request_resolution_serializer.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyRequestResolutionSerializer
+    include JSONAPI::Serializer
+
+    attribute :id
+
+    attribute :type do |object|
+      object.resolving_type.demodulize.underscore.split('_').last
+    end
+
+    attribute :decision_type, if: proc { |obj| obj.resolving.respond_to?(:type) } do |object|
+      object.resolving.type
+    end
+
+    attribute :reason
```

Comment:
Reason will only need to be serialized on a declination in the decision serializer variant.